### ISSUE TITLE
options.instanceId doesn't support Symbols as instance name.

### ISSUE THREAD

**Issue by senadir** [zalmoxisus/redux-devtools-extension#826]
Initiating `__REDUX_DEVTOOLS_EXTENSION__.options.instanceId` with a symbol would throw an error.

```
Uncaught DOMException: Failed to execute 'postMessage' on 'Window': Symbol(wc/store/cart) could not be cloned.
    at post (<anonymous>:13794:10)
    at toContentScript (<anonymous>:13885:3)
    at relay (<anonymous>:14240:5)
    at init (<anonymous>:14440:5)
```

Use case:
I'm using redux devtools alongside [`@wordpress/data`](https://github.com/WordPress/gutenberg/tree/trunk/packages/data), which is a superset of redux that supports multiple stores along other things.
 `@wordpress/data` register each store with `__REDUX_DEVTOOLS_EXTENSION__`, using [the store name as the name and instanceId for the plugin](https://github.com/WordPress/gutenberg/blob/5dbf7ca8a285f5cab65ebf7ab87dafeb6118b6aa/packages/data/src/redux-store/index.js#L242-L252).

![image](https://user-images.githubusercontent.com/6165348/146063672-efb8ce20-4d05-482e-8c61-cfe189479eef.png)
 
Reproduce steps:
You can see this example https://codesandbox.io/s/silent-silence-fknrn?file=/src/index.js

Basically:
```js
import { createStore } from "redux";
import { devToolsEnhancer } from "redux-devtools-extension";

const store = createStore(
  () => {},
  devToolsEnhancer({
    instanceId: Symbol("test")
  })
);
```

Possible fixes:
- Detect if instanceId is a symbol `typeof instanceId === 'symbol'` and serialize it `instanceId.toString()` which would give you `'Symbol("test")'`.

**Comment by alexflorisca** [comment]
Just pinging to see if there's any chance of getting this merged any time soon?
